### PR TITLE
Add gmailr 1.0.0 post

### PR DESCRIPTION
**Pull request by jimhester**
@jenny is on vacation, so I don't know if she has time to review it, if not that is fine, I don't think there is anything major that needs to change.

### CODE REVIEW

File: content/articles/2019-08-gmailr-1-0-0.Rmarkdown [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+The new strategy makes it harder to accidentally push your tokens to the cloud, easier to use multiple Google identities, and easier to share tokens across projects and packages.
+
+Overall, gmailr has gotten more careful about getting your permission to use a cached token. See the gargle vignette [Non-interactive auth](https://gargle.r-lib.org/articles/non-interactive-auth.html) to learn how to prevent attempts to interact with you, especially the section ["Arrange for an oauth token to be re-discovered"](https://gargle.r-lib.org/articles/non-interactive-auth.html#arrange-for-an-oauth-token-to-be-re-discovered).
+
+gmailr also removes the built-in OAuth "app". This was nessesariy to comply with stricter enforncement of the Google terms of service. It was not possible to comply with the TOS and continue to have a default application embedded in gmailr. See the [setup](http://gmailr.r-lib.org/#setup) section of the readme for how to create a new project and authenticate it with only a few steps.
```
Comment (jennybc) [inline content/articles/2019-08-gmailr-1-0-0.Rmarkdown]:
```suggestion
gmailr also removes the built-in OAuth "app". This was nessesary to comply with stricter enforcement of the Google terms of service. It was not possible to comply with the TOS and continue to have a default application embedded in gmailr. See the [setup](http://gmailr.r-lib.org/#setup) section of the readme for how to create a new project and authenticate it with only a few steps.
```

Comment (jennybc) [reply content/articles/2019-08-gmailr-1-0-0.Rmarkdown]: I'm not sure I would characterize this as a TOS issue. I'd probably say that there are special new rules around the Gmail API that make it extremely difficult to get a Gmail-capable app verified and there is little point in shipping an app that is unverified.

Comment (jimhester) [reply content/articles/2019-08-gmailr-1-0-0.Rmarkdown]: Ok, I tweaked the wording and added a link to the google documentation on the subject.

**Comment by jimhester** [comment]
I forgot this wasn't merged, whoops!

**Pull request merged**